apps/service-portal/src/components/Layout/Layout.css.ts

```diff
@@ -8,7 +8,7 @@ export const layoutWrapper = style({
 
 export const layoutGrid = style({
   transition: 'margin 150ms ease-in-out, flexBasis 150ms ease-in-out',
-  willChange: 'transition',
+  willChange: 'margin, flexBasis',
```

Comment:
This goes for the `transitions` prop as well ... not sure if I'm correct or not.

But I assume that values should follow CSS naming conventions (kebab) ... so this needs to be `flex-basis`.
Did the other one work?

Reply:
lord help me, haha fixed 😅